addons/hr_expense/models/hr_expense.py

```diff
@@ -310,9 +359,11 @@ def create_expense_from_attachments(self, attachment_ids=None, view_type='tree')
 
         for attachment in attachments:
             expense = self.env['hr.expense'].create({
-                'name': attachment.name.split('.')[0],
+                'name': _('product'),
```

Comment:
I'm not sure I understand this change, but why no caps?

Reply:
It was in spec to set Description =  product, for the expenses that are created by/from scanning

Reply:
Ho please!
What the spec mean is
```suggestion
                'name': product.name,
```